### DISCUSSION TITLE
How to properly cite this work?

### DISCUSSION THREAD

**Discussion by sbowman-mitre** [TUDelft-CNS-ATM/openap#9] [General]
I'm working on a white paper that will need to cite this repository. I noticed that the README for [WRAP](https://github.com/junzis/wrap) contains a helpful recommendation for properly citing that work, but this repository does not. Do you have a doi for this work and can you recommend a citation methodology? Thanks!

**Comment by junzis** [comment]
Hi @sbowman-mitre, I added the citation info now: https://github.com/junzis/openap#citing-openap
Cheers!

**Reply by sbowman-mitre** [reply]
Thanks!
